calcom/cal.com
# feat: admin org list now shows all, dns check added

## What does this PR do?

Administrator Organization list now shows all organizations, verified or not, and a DNS check was added to flag which organizations successfully registered a subdomain in a Vercel hosting environment or which were not configured at all, sending an email notifications to the Cal.com instance admins.

Here is the list of orgs for admins:
<kbd><img src="https://github.com/calcom/cal.com/assets/467258/c7bafdd1-7d2a-4dd1-9cda-e8537dff9bb2" /></kbd>

Verification actions:
<kbd><img src="https://github.com/calcom/cal.com/assets/467258/d70c629d-9894-45cf-b010-c43354b0b50f" /></kbd>

Verification actions plus mark DNS as configured:
<kbd><img src="https://github.com/calcom/cal.com/assets/467258/bc9fd646-b42c-4b18-a355-ee13d1297092" /></kbd>

Email sent to instance admins:
<kbd><img src="https://github.com/calcom/cal.com/assets/467258/11e9576e-3743-4337-aa04-e4577e43bf99" /></kbd>

## Type of change


## How should this be tested?

Create a bunch of orgs in a non-Vercel environment (such as localhost) and see in the Organizations Admin Settings all the created orgs. Verify some of them and you will still see them in the list without the badge Unverified. Mark some of them as DNS configured and see them in the list without the DNS missing badge.
